hw2/utils.ts

```diff
@@ -0,0 +1,72 @@
+import { User } from './models/user.model';
+
+let usersList: User[] = [
+    {
+        id: '1',
+        login: 'Abc',
+        password: '123',
+        age: 1,
+        isDeleted: false
+    },
+    {
+        id: '2',
+        login: 'Bc',
+        password: '456',
+        age: 2,
+        isDeleted: false
+    },
+    {
+        id: '3',
+        login: 'Cb',
+        password: '789',
+        age: 3,
+        isDeleted: false
+    }
+];
+
```

Comment:
Think about making `usersList` an object with `key` as user id and `value` as user data.
```
const usersStore: Record<string, User> = {
```
It would improve the performance and overall experience of work with user data.